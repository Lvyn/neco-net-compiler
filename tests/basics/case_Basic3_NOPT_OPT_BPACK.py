from snakes.nets import *

net = PetriNet('Net')
net.processes = []

s1 = Place('s1', [dot], tBlackToken)
s1.flow_control = False
s1.one_safe = True
s1.process_name = None

s2 = Place('s2', [], tBlackToken)
s2.flow_control = False
s2.one_safe = True
s2.process_name = None

s3 = Place('s3', [], tBlackToken)
s3.flow_control = False
s3.one_safe = True
s3.process_name = None


net.add_place(s1)
net.add_place(s2)
net.add_place(s3)

transition = Transition('t1', Expression('True'))
net.add_transition(transition)

net.add_input('s1', 't1', Value(dot))
net.add_output('s2', 't1', Value(dot))

transition = Transition('t2', Expression('True'))
net.add_transition(transition)

net.add_input('s2', 't2', Value(dot))
net.add_output('s3', 't2', Value(dot))
