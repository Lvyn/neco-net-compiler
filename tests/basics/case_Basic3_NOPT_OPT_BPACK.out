[{
's1' : [dot, ], 
's2' : [], 
's3' : [], 
}, {
's1' : [], 
's2' : [dot, ], 
's3' : [], 
}, {
's1' : [], 
's2' : [], 
's3' : [dot, ], 
}, ]